src/HTMLGenerator/converter.py

```diff
@@ -60,3 +61,57 @@ def convert(top_level_output_directory, top_level_input_directory, json_with_fil
     os.remove(current_configuration_absolute_path)
 
     return
+
+def convertFromCli(jsonFilePath) :
+    with open(jsonFilePath) as jsonFile:
+            json_with_filepaths = json.load(jsonFile)
+            converter_path = Path(Path("C:/FINALE/src/HTMLGenerator") / "ConvertFromConfigurationFile.vi")
```

Comment:
There shouldn't be such absolute paths in the code, this is very error prone. There's no way you can predict that every single user will use this path for the FINALE git repo, so this is very likely to fail in someone else's machine.

Reply:
Removed